statsmodels/tsa/statespace/tests/test_kalman.py

```diff
@@ -675,4 +676,34 @@ def test_filtered_state(self):
         assert_almost_equal(
             self.result['state'][5][-1],
             self.true_states.iloc[end-1, 3], 4
-        )
\ No newline at end of file
+        )
+
+
+def test_standardized_forecasts_error():
+    """
+    Simple test that standardized forecasts errors are calculated correctly.
+
+    Just uses a different calculation method on a univariate series.
+    """
+
+    # Get the dataset
+    true = results_kalman_filter.uc_uni
+    data = pd.DataFrame(
+        true['data'],
+        index=pd.date_range('1947-01-01', '1995-07-01', freq='QS'),
+        columns=['GDP']
+    )
```

Comment:
I don't really like the closing parenthesis in front. I would prefer it to be below the `c` of columns